pkg/minishift/constants/constants.go

```diff
@@ -37,10 +37,12 @@ const (
 	SkipVerifyInsecureTLS       = "insecure-skip-tls-verify=true"
 	BinaryName                  = "minishift"
 	OcPathInsideVM              = "/var/lib/minishift/bin"
+	BaseDirInsideInstance       = "/var/lib/minishift/base"
 )
 
 var (
 	ValidIsoAliases = []string{B2dIsoAlias, CentOsIsoAlias}
+	ValidComponents = []string{"automation-service-broker", "service-catalog", "template-service-broker"}
```

Comment:
Instead of parsing I put the valid component here because it may happen user just download the minishift and then list the component and if oc binary not available then you never know the what components are vaild, also for `m openshift component list` having oc available shouldn't be required. I put simple documentations so that when there is any component added/removed we just need to change here instead in documents.